iocore/net/quic/QUICStream.cc

```diff
@@ -297,13 +296,7 @@ QUICStream::_write_to_read_vio(QUICOffset offset, const uint8_t *data, uint64_t
 void
 QUICStream::on_read()
 {
-  this->_state.update_on_read();
-}
-
-void
-QUICStream::on_eos()
-{
-  this->_state.update_on_eos();
+  this->_state.update_on_user_read_event();
```

Comment:
`QUICStream::on_read` now process user event. That transit QUICReceiveState to terminal states (ResetRead or DataRead).